source/wp-content/themes/wporg-parent-2021/patterns/heading-with-arrow-and-button.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Title: Heading with arrow and button
+ * Slug: wporg-parent-2021/heading-with-arrow-and-button
+ * Categories: header
+ */
+
+?>
+<!-- wp:group {"align":"full","backgroundColor":"blueberry-1"} -->
+<div class="wp-block-group alignfull has-blueberry-1-background-color has-background">
+	<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"97px","bottom":"97px","right":"80px","left":"80px"}}},"className":"heading-with-arrow","layout":{"type":"flex","orientation":"vertical","justifyContent":"center","flexWrap":"nowrap"}} -->
+	<div class="wp-block-group alignfull heading-with-arrow" style="padding-top:97px;padding-right:80px;padding-bottom:97px;padding-left:80px">
+		<!-- wp:group {"style":{"spacing":{"blockGap":"0px"}},"layout":{"inherit":false}} -->
+		<div class="wp-block-group">
+			<!-- wp:group {"style":{"spacing":{"padding":{"bottom":"67px"}}},"className":"heading-with-arrow__row","layout":{"type":"flex","flexWrap":"nowrap","orientation":"horizontal","justifyContent":"center","verticalAlignment":"top"}} -->
+			<div class="wp-block-group heading-with-arrow__row" style="padding-bottom:67px">
+				<!-- wp:group {"layout":{"contentSize":"760px","inherit":false}} -->
+				<div class="wp-block-group">
+					<!-- wp:paragraph {"style":{"typography":{"fontSize":"120px","fontStyle":"normal","fontWeight":"300","lineHeight":"1.1","letterSpacing":"-0.02em"}},"textColor":"white"} -->
+					<p class="has-white-color has-text-color" style="font-size:120px;font-style:normal;font-weight:300;letter-spacing:-0.02em;line-height:1.1">We'll help you get started</p>
+					<!-- /wp:paragraph -->
+				</div>
+				<!-- /wp:group -->
+
+				<!-- wp:image {"id":8960,"width":370,"height":120,"sizeSlug":"full","linkDestination":"none","style":{"spacing":{"margin":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}}}} -->
+				<figure class="wp-block-image size-full is-resized" style="margin-top:0px;margin-right:0px;margin-bottom:0px;margin-left:0px"><img src="https://wordpress.org/main-test/files/2022/07/Arrow-3.png" alt="" class="wp-image-8960" width="370" height="120"/></figure>
```

Comment:
I think this would be better as a background image rather than content. News uses a pseudo element for this:

![Screen Shot 2022-08-02 at 1 43 37 PM](https://user-images.githubusercontent.com/1017872/182273965-d7616889-001a-4842-9fb6-5b22906996ca.jpg)

Reply:
No, the arrow doesn't need to be editable. You can use the same method as News.